From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Antonino Orlando <orlando.antonino@gmail.com>
Date: Sun, 29 Jun 2025 12:58:18 +0200
Subject: [PATCH 2/3] mediatek: add support for Zyxel NWA90AX PRO

Same hardware as Zyxel NWA50AX PRO.
DEVICE_ALT0_VENDOR := Zyxel
DEVICE_ALT0_MODEL := NWA90AX Pro

Signed-off-by: Antonino Orlando <orlando.antonino@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/19257
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

(cherry picked from commit e34e874a11f0506ff17365819bc101a455dc503e)
[backport to openwrt-24.10: unchanged]
---
 target/linux/mediatek/image/filogic.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/target/linux/mediatek/image/filogic.mk b/target/linux/mediatek/image/filogic.mk
index b7f8ffd47c..97589ec160 100644
--- a/target/linux/mediatek/image/filogic.mk
+++ b/target/linux/mediatek/image/filogic.mk
@@ -2230,6 +2230,8 @@ TARGET_DEVICES += zyxel_ex5700-telenor
 define Device/zyxel_nwa50ax-pro
   DEVICE_VENDOR := Zyxel
   DEVICE_MODEL := NWA50AX Pro
+  DEVICE_ALT0_VENDOR := Zyxel
+  DEVICE_ALT0_MODEL := NWA90AX Pro
   DEVICE_DTS := mt7981b-zyxel-nwa50ax-pro
   DEVICE_DTS_DIR := ../dts
   DEVICE_PACKAGES := kmod-mt7915e kmod-mt7981-firmware mt7981-wo-firmware zyxel-bootconfig
